package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "safe-airdrop",
   "version": "1.2.0",
+  "license": "MIT",
```

Comment:
I was trying to get around this annoying warning, but it doesn't seem to help...

```
warning ../../../package.json: No license field
```